src/main/java/com/github/manolo8/darkbot/core/objects/Clickable.java

```diff
@@ -40,7 +47,8 @@ public void reset() {
      * @return prevent swf crash
      */
     private boolean isInvalid() {
-        return defRadius <= 0 || address == 0 || API.readMemoryLong(address) != confirm;
+        return address == 0
+               || API.readLong(address + 32) != owner.address; // confirm owner address
```

Comment:
```suggestion
        return defRadius <= 0 || address == 0 || API.readLong(address + 32) != owner.address; // confirm owner address
```
In this case, even if redundant, i would rather keep the def radius check. I've seen too many weird things happen with this logic and i don't want random crashes to occur for something so easy to keep

Reply:
Sometimes, no idea why, the entities(mostly boxes) have `defRadius == 0`, and bot is unable to select it cause this method returns true

Reply:
precisely, if the radius is 0 for odd reasons we don't want to mess with them, gotta find where the real bug is and fix it